src/library/dlgcoverartfullsize.h

```diff
@@ -37,7 +37,5 @@ class DlgCoverArtFullSize
     QPixmap m_pixmap;
     TrackPointer m_pLoadedTrack;
     BaseTrackPlayer* m_pPlayer;
-    WCoverArtMenu* m_pCoverMenu;
+    QPointer<WCoverArtMenu> m_pCoverMenu;
```

Comment:
Why we need a QPointer here?
I think parented_ptr is appropriated.

Reply:
Why parented_ptr if Qt offers QPointer for this purpose? I still don't understand why we need parented_ptr.

Reply:
QPointer has a different purpose.
It is a kind of weak pointer to an object owned by the Qt object tree.

If you have a pointer of this type, you can check for null before every access to use it in a safe way in a sibiling object relative to the owner for instance.

We have original introduced the patented pointer to document the ownership of the parent and force this at the same time. QPointer does not have this feature.